packages/ajax/stories/index.stories.mdx

```diff
@@ -0,0 +1,164 @@
+      Execute Request to Action Logger
+    </button>
+  `}
+</Story>
+
+```js
+ajax.get('./packages/ajax/stories/data.json').then(response => console.log(response.data));
+```
+
+## Features
+
+- only JS functions, no (unnecessarily expensive) web components
+- supports GET, POST, PUT, DELETE, REQUEST, PATCH and HEAD methods
+- can be used with or without XSRF token
+
+## How to use
+
+### Installation
+
+```sh
+npm i --save @lion/ajax
+```
+
+```js
+import { ajax, AjaxClass } from '@lion/ajax';
+```
+
+## Cancelable Request
+
+It is possible to make an Ajax request cancelable, and then call `cancel()` to make the request provide a custom error once fired.
+
+<Story name="Cancelable">
+  {html`
+    <button
+      @click=${() => {
+        const myAjax = AjaxClass.getNewInstance({ cancelable: true });
+        requestAnimationFrame(() => {
+          myAjax.cancel('too slow');
+        });
+        myAjax
+          .get('./packages/ajax/stories/data.json')
+          .then(response => {
+            console.log(response.data);
+          })
+          .catch(error => {
+            console.log(error);
+          });
+      }}
+    >
+      Execute Request to Action Logger
+    </button>
+  `}
+</Story>
+
+```js
+const myAjax = AjaxClass.getNewInstance({ cancelable: true });
+
+requestAnimationFrame(() => {
+  myAjax.cancel('too slow');
+});
+
+myAjax
+  .get('./packages/ajax/stories/data.json')
+  .then(response => {
+    console.log(response.data);
+  })
+  .catch(error => {
+    console.log(error);
+  });
+```
+
+## Cancel concurrent requests
+
+You can cancel concurrent requests with the `cancelPreviousOnNewRequest` option.
+
+<Story name="Cancel Previous on New Request">
+  {html`
+    <button
+      @click=${() => {
+        const myAjax = AjaxClass.getNewInstance({ cancelPreviousOnNewRequest: true });
+        myAjax
+          .get('./packages/ajax/stories/data.json')
+          .then(response => {
+            console.log(response.data);
+          })
+          .catch(error => {
+            console.log(error.message);
+          });
+        myAjax
+          .get('./packages/ajax/stories/data.json')
+          .then(response => {
+            console.log(response.data);
+          })
+          .catch(error => {
+            console.log(error.message);
+          });
+      }}
+    >
+      Execute Both Requests to Action Logger
+    </button>
+  `}
+</Story>
+
+```js
+const myAjax = AjaxClass.getNewInstance({ cancelPreviousOnNewRequest: true });
+
+myAjax
+  .get('./packages/ajax/stories/data.json')
+  .then(response => {
+    console.log(response.data);
+  })
+  .catch(error => {
+    console.log(error.message);
+  });
+
+myAjax
+  .get('./packages/ajax/stories/data.json')
+  .then(response => {
+    console.log(response.data);
+  })
+  .catch(error => {
+    console.log(error.message);
+  });
+```
+
+## Considerations
+
+Due to a [bug in axios](https://github.com/axios/axios/issues/385) options may leak in to other instances.
```

Comment:
Links are rendered weird:

```html
<a href="http://localhost:9002https://github.com/axios/axios/issues/385" class="sbdocs sbdocs-a css-1cvpre6">bug in axios</a>
```

Reply:
Odd.. will need to look into how to prevent that in storybook .mdx files

Reply:
https://github.com/storybookjs/storybook/issues/9005 I think this is related